src/api/endpoints/activities/activities.py

```diff
@@ -19,6 +23,7 @@ def __init__(self, **kwargs):
     @roles_required(["success ops", "society president"])
     def post(self):
         """Create an activity."""
+        from manage import app
```

Comment:
is it possible to inject this in the constructor of the view?

Reply:
Hi @georgreen I tried implementing this approach but it fails to execute when injected into the constructor class. Can I go ahead and merge the PR as is?

Reply:
@kevinbett Sure, just add a TODO comment explaining `why we have that and we should try to remove it there in future refactors.` Then you can proceed to merge.

Reply:
Okay sure.